scram-tools.file/bin/get_tools

```diff
@@ -12,7 +12,10 @@ export SCRAM_TOOL_SOURCE_DIR=$(realpath ${SCRAM_TOOLS_BIN_DIR}/../tools)/$4
 
 #setup tools specific env
 tool_script=${SCRAM_TOOL_SOURCE_DIR}/env.sh
-if [ -f $tool_script ]; then source $tool_script; fi
+if [ -f $tool_script ]; then
+  perl -p -i -e 's|\@([^@]*)\@|$ENV{$1}|g' $tool_script
+  source $tool_script
+fi
```

Comment:
I think this is not needed if `scram-tools.file/tools/gcc/env.sh` uses `CMS_CXX_STANDARD` env directly

Reply:
I wasn't sure if this env variable will (still) be defined at that point. I will change this in the next iteration.

Reply:
it should be defined. Just like `COMPILER_CXXFLAGS`  which is set https://github.com/cms-sw/cmsdist/pull/8558/files#diff-9ae93e4d39c78dbb82f952da0248bf67d11ce362f51e329364779da933923564R12  and used by https://github.com/cms-sw/cmsdist/pull/8558/files#diff-09500a932b32fbe40198bc05cbf2ebbb173183365f9a4c751795eb25ac9c7ddeR62